talk/setup.tex

```diff
@@ -189,10 +187,38 @@
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 % easy links to godbolt and cppinsight %
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\newcommand\listofgodbolt{%
+  \addtocontents{lgb}{\end{enumerate}}
+  \scriptsize
+  \begin{multicols}{2}
+    \@starttoc{lgb}%
+  \end{multicols}
+}
+\AtBeginDocument{ \addtocontents{lgb}{\begin{enumerate} } }
+
+\newcounter{gb@counter}
+\regtotcounter{gb@counter}
+\resetcounteronoverlays{gb@counter}
+\newcommand*{\overlaynumber}{\number\beamer@slideinframe}
+
 % Use as \godbolLink{url}
-\newcommand\godboltLink[1]{%
-  \href{#1}{\color{blue!50!white} godbolt}%
+\newcommand\godboltLink[2]{%
+  \ifnum \overlaynumber=1%
+    \href{#1}{\color{blue!50!white} godbolt}%
```

Comment:
```suggestion
  \href{#1}{\color{blue!50!white} godbolt}%
    \ifnum \overlaynumber=1%
```
Like this, the link will also show on the other slides, but the entry to the index only happens once.
However, if the box only appears on overlay != 1, there won't be an entry at all. Is this an issue?

Reply:
Oh indeed ! Thanks for spotting.